integrations/cross_platform/yocto.rst

```diff
@@ -0,0 +1,369 @@
+
+Set up the Conan layer
+----------------------
+
+We have created a `meta-conan <https://github.com/conan-io/meta-conan>`_ layer that includes all the configuration, the Conan client and a
+generic BitBake recipe. To add the layer you will have to clone the repository and the dependency layers of ``meta-openembedded``:
+
+.. code-block:: bash
+
+    $ cd poky
+    $ git clone https://github.com/conan-io/meta-conan.git
+    $ git clone --branch thud https://github.com/openembedded/meta-openembedded.git
+
+You would also have to activate the layers in the *bblayers.conf* file of your build folder:
+
+.. code-block:: text
+   :caption: *conf/bblayers.conf*
+
+    POKY_BBLAYERS_CONF_VERSION = "2"
+
+    BBPATH = "${TOPDIR}"
+    BBFILES ?= ""
+
+    BBLAYERS ?= " \
+    /home/username/poky/meta \
+    /home/username/poky/meta-poky \
+    /home/username/poky/meta-yocto-bsp \
+    /home/username/poky/meta-openembedded/meta-oe \
+    /home/username/poky/meta-openembedded/meta-python \
+    /home/username/poky/meta-conan \
+    "
+
+.. note::
+
+    Currently there is no support for ``CONAN_REVISIONS_ENABLED``, so remote and virtual Artifactory repositories will not work in this
+    case. We will continue working on this layer to support more features.
+
+    Please report any question, feature request or issue related to the ``meta-conan`` layer in its
+    `GitHub issue tracker <https://github.com/conan-io/meta-conan/issues>`_.
+
+Write the Bitbake recipe for the Conan package
+----------------------------------------------
+
+With the ``meta-conan`` layer, a Conan recipe to deploy a Conan package should look as easy as this recipe:
+
+.. code-block:: text
+   :caption: *conan-mosquitto_1.4.15.bb*
+
+    inherit conan
+
+    DESCRIPTION = "An open source MQTT broker"
+    LICENSE = "EPL-1.0"
+
+    CONAN_PKG = "mosquitto/1.4.15@bincrafters/stable"
+
+This recipe will be placed inside your application layer that should be also added to the *conf/bblayers.conf* file.
+
+Configure Conan variables for the build
+---------------------------------------
+
+Additionally to the recipe, you will need to provide the information about the credentials for Artifactory or the profile to be used to
+retrieve the packages in the *local.conf* file of your build folder.
+
+.. code-block:: text
+   :caption: *poky_build_folder/conf/local.conf*
+
+    IMAGE_INSTALL_append = " conan-mosquitto"
+
+    # Profile for installation
+    CONAN_PROFILE_PATH = "${TOPDIR}/conf/armv8"
+    # Artifactory repository
+    CONAN_REMOTE_URL = "https://localhost:8081/artifactory/api/conan/<repository>"
+    # Artifactory Credentials
+    CONAN_USER = "REPO_USER"
+    CONAN_PASSWORD = "REPO_PASSWORD"
+
+Notice the *armv8* profile to indicate your configuration next to the *local.conf*. That way you will be able to match the Conan
+configuration with the specific architecture or board of your Yocto build.
+
+.. code-block:: text
```

Comment:
this profile is copied exactly the same above? Is it necessary to repeat it?

Reply:
It was a previous recommendation from a review. I think it could clarify readers the location of the profile, but just that